src/Microsoft.Azure.WebJobs.ServiceBus/ServiceBusAttribute.cs

```diff
@@ -38,6 +38,18 @@ public sealed class ServiceBusAttribute : Attribute, IConnectionProvider
         public ServiceBusAttribute(string queueOrTopicName)
         {
             QueueOrTopicName = queueOrTopicName;
+            EntityType = EntityType.Queue;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ServiceBusAttribute"/> class.
+        /// </summary>
+        /// <param name="queueOrTopicName">The name of the queue or topic to bind to.</param>
+        /// <param name="queueOrTopicName">The type of the entity to bind to.</param>
+        public ServiceBusAttribute(string queueOrTopicName, EntityType entityType)
```

Comment:
curious: why do we need to move this to a ctor parameter?

Reply:
because we added: `metadata["entityType"] = "Queue";` in this change. In V1 EntityType property is set up in script repo.